src/sbs/core.py

```diff
@@ -0,0 +1,2 @@
+
+NUM_PAT = "[+-]?[0-9]*[.]?[0-9]+"  # Regex pattern for any number, positive or negative, int or float.
```

Comment:
For now, please add this to the top of acs.py instead of importing from a new core module